src/components/project/list.vue

```diff
@@ -244,6 +244,7 @@ export default {
   "en": {
     "heading": [
       "Welcome to Central.",
+      // This is a tagline displayed next to "Welcome to Central". The text adds visual balance to the page and does not need to be translated literally to your language. The text should be action-oriented, collaborative, and a little bit fun.
```

Comment:
No need if you prefer a single line, but just FYI, `restructure.js` does lets you split a comment over multiple lines.

Reply:
Good to know, thanks!